apis/v1alpha1/forwarding_target_types.go

```diff
@@ -139,7 +167,8 @@ type GenericForwardToTarget struct {
 	// Support: Core (HTTPRoute)
 	// Support: Extended (TCPRoute)
 	//
-	// +optional
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=100
```

Comment:
I am not sure we should have a max here, the sum could be greater than 100. At least in theory we can have 
```
weight=1
weight=999
```

Unless we are intentionally restricting the granularity

Reply:
Is 1000 a sufficient level of granularity here? I'd initially thought 100 would be sufficient, but you're right that we can have a max variation 100:1, and maybe some users will want to split off a tinier fraction than that?

Reply:
Max: 10000

Reply:
With a ratio of 10000:1, one can get a split of 0.01%, which seems to be enough for most use cases I know of. In case this is not sufficient, we can again bump in future.

Reply:
Updated, thanks!